src/plugins/explore/public/application/pages/traces/trace_details/public/traces/span_tabs/span_overview_tab.tsx

```diff
@@ -94,15 +95,16 @@ export const SpanOverviewTab: React.FC<SpanOverviewTabProps> = ({
     }
 
     const spanId = selectedSpan.spanId;
-    const serviceName = selectedSpan.serviceName;
+    const serviceName = resolveServiceNameFromSpan(selectedSpan);
     const operation = selectedSpan.name;
     const duration = extractSpanDuration(selectedSpan);
     const startTime = selectedSpan.startTime;
-    const hasError = selectedSpan['status.code'] === 2;
+    const hasError = isSpanError(selectedSpan);
 
     // Extract HTTP-specific attributes
-    const httpMethod = selectedSpan.attributes?.['http.method'];
-    const httpUrl = selectedSpan.attributes?.['http.url'];
+    const httpMethod =
+      selectedSpan.attributes?.['http.method'] || selectedSpan.attributes?.['http.request.method'];
+    const httpUrl = selectedSpan.attributes?.['http.url'] || selectedSpan.attributes?.['url.full'];
```

Comment:
nit. Ideally these strings should be moved to some constants.

Reply:
Temporary, if this is maintained I will create constants in follow-up.